climakitae/core/data_load.py

```diff
@@ -737,406 +795,421 @@ def _check_valid_unit_selection(selections):
+
+    # Load mixing ratio data
+    selections.variable_id = ["q2"]
+    selections.units = "g kg-1"
+    q2_da = _get_data_one_var(selections)
+
+    # Load pressure data
+    selections.variable_id = ["psfc"]
+    selections.units = "hPa"
+    pressure_da = _get_data_one_var(selections)
+
+    # Load u10 data
+    selections.variable_id = ["u10"]
+    selections.units = (
+        "m s-1"  # Need to set units to required units for compute_wind_mag
+    )
+    u10_da = _get_data_one_var(selections)
+
+    # Load v10 data
+    selections.variable_id = ["v10"]
+    selections.units = "m s-1"
+    v10_da = _get_data_one_var(selections)
+
+    # Derive relative humidity
+    # Returned in units of [0-100]
+    rh_da = compute_relative_humidity(
+        pressure=pressure_da,  # hPa
+        temperature=t2_da_C,  # degC
+        mixing_ratio=q2_da,  # g/kg
+    )
 
-        # Hard set timescale to hourly
-        orig_timescale = selections.timescale  # Preserve original user selection
-        selections.timescale = "hourly"
-
-        # Load temperature data
-        selections.variable_id = ["t2"]
-        selections.units = "degC"  # Kelvin required for humidity
-        t2_da_C = _get_data_one_var(selections)
-
-        # Load mixing ratio data
-        selections.variable_id = ["q2"]
-        selections.units = "g kg-1"
-        q2_da = _get_data_one_var(selections)
-
-        # Load pressure data
-        selections.variable_id = ["psfc"]
-        selections.units = "hPa"
-        pressure_da = _get_data_one_var(selections)
-
-        # Load u10 data
-        selections.variable_id = ["u10"]
-        selections.units = (
-            "m s-1"  # Need to set units to required units for compute_wind_mag
-        )
-        u10_da = _get_data_one_var(selections)
-
-        # Load v10 data
-        selections.variable_id = ["v10"]
-        selections.units = "m s-1"
-        v10_da = _get_data_one_var(selections)
-
-        # Derive relative humidity
-        # Returned in units of [0-100]
-        rh_da = compute_relative_humidity(
-            pressure=pressure_da,  # hPa
-            temperature=t2_da_C,  # degC
-            mixing_ratio=q2_da,  # g/kg
-        )
+    # Derive windspeed
+    # Returned in units of m/s
+    windspeed_da_ms = compute_wind_mag(u10=u10_da, v10=v10_da)  # m/s
 
-        # Derive windspeed
-        # Returned in units of m/s
-        windspeed_da_ms = compute_wind_mag(u10=u10_da, v10=v10_da)  # m/s
+    # Convert units to proper units for fosberg index
+    t2_da_F = convert_units(t2_da_C, "degF")
+    windspeed_da_mph = convert_units(windspeed_da_ms, "mph")
 
-        # Convert units to proper units for fosberg index
-        t2_da_F = convert_units(t2_da_C, "degF")
-        windspeed_da_mph = convert_units(windspeed_da_ms, "mph")
+    # Compute the index
+    da = fosberg_fire_index(
+        t2_F=t2_da_F, rh_percent=rh_da, windspeed_mph=windspeed_da_mph
+    )
+
+    return da
```

Comment:
I'm guessing that these function edits are because git tracked them weirdly, not that there are any changes to these functions within this PR, right?

Reply:
yeah exactly